lib/IDE/CodeCompletionResultPrinter.cpp

```diff
@@ -120,6 +120,9 @@ class AnnotatingResultPrinter {
     OS << content.substr(rtrimIdx);
   }
 
+  /// Print an empty \p tag.
+  void printEmptyTag(StringRef tag) { OS << "<" << tag << " />"; }
+
```

Comment:
This is unused.

Reply:
Thanks! I'll let tests pass and then remove + smoke check.